pychromecast/controllers/plex.py

```diff
@@ -166,8 +181,18 @@ def media_to_chromecast_command(
     return msg
 
 
-@property
-def episode_title(self):
+class PlexMediaStatus(MediaStatus):
+    """Class to hold the media status."""
+
+    @property
+    def episode_title(self) -> str | None:
+        """Return episode title."""
+        return self.media_metadata.get("subtitle")
+
+
+# The episode_title property is added to MediaStatus objects
+@property  # type: ignore[misc]
+def episode_title(self: PlexMediaStatus) -> str | None:
```

Comment:
This is weird. I'd make this a regular function if it's needed to be separate from the class.

Reply:
Yes, it's very weird, I didn't want to clean it up in this PR though